lib/logstash/outputs/rabbitmq.rb

```diff
@@ -15,6 +18,8 @@ class RabbitMQ < LogStash::Outputs::Base
       include LogStash::PluginMixins::RabbitMQConnection
 
       config_name "rabbitmq"
+      
+      default :codec, "json"
```

Comment:
Ah, the file input does it the way I do it here. Do you just mean I should add comments above?